tests/rptest/services/redpanda.py

```diff
@@ -1568,15 +1587,10 @@ def healthy(self):
         the health of a node after a restart.
         """
         counts = {self.idx(node): None for node in self.nodes}
+        count_query = f"select sum(value) as s from metrics where name = 'vectorized_cluster_partition_under_replicated_replicas'"
         for node in self.nodes:
-            metrics = self.metrics(node)
             idx = self.idx(node)
-            for family in metrics:
-                for sample in family.samples:
-                    if sample.name == "vectorized_cluster_partition_under_replicated_replicas":
-                        if counts[idx] is None:
-                            counts[idx] = 0
-                        counts[idx] += int(sample.value)
+            counts[idx] = self.query_metrics(node, count_query).at[0, 's'] # None if metric is not populated, sum(value) otherwise.
```

Comment:
The combination of a sum query and a `.at[0...` on the result seems like it could do with wrapping in a friendly python method so that people writing tests don't have to learn the query language.

Reply:
Ack, will do.